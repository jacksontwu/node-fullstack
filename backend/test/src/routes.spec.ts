import AppRoutes from '../../src/routes';

describe('AppRoutes', () => {
    it('routes', () => {
        expect(AppRoutes).toMatchSnapshot();
    });
});
