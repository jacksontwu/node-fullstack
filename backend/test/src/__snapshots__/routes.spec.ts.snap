// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppRoutes routes 1`] = `
Array [
  Object {
    "action": [Function],
    "method": "get",
    "path": "/health",
  },
]
`;
